### ISSUE TITLE
LIGO code-along

### ISSUE THREAD

**Issue by betatim** [CERNStudyGroup/cernstudygroup.github.io#14]
Date: 26 Feb 2016?
Lesson guide: TBD
Lesson materials: ___

---

We should have an event where we rediscover gravity waves. Their data is up: https://losc.ligo.org/events/GW150914/ their stack is python so CERN should be able to confirm what these new kids on the block are doing :stuck_out_tongue_winking_eye: 

:ocean::ocean::ocean: or :wavy_dash::earth_africa::wavy_dash:?

**Comment by adavidzh** [comment]
+1

**Comment by RaoOfPhysics** [comment]
+1

**Comment by betatim** [comment]
Does anyone know a LIGO person who has more insight than us "decoding" the [jupyter notebook](https://losc.ligo.org/s/events/GW150914/GW150914_tutorial.html) they posted that explains some of the analysis?

**Comment by suenjedt** [comment]
Super nice idea. I will ask around if someone knows anyone there. I dont.

**Comment by tpmccauley** [comment]
I love what they've done regarding the jupyter notebook. If only there were more hours in the day...

**Comment by RaoOfPhysics** [comment]
@tpmccauley: You should spend your limited time fiddling with the Jupyter notebook instead of reading their published scientific papers. :P

**Comment by ibab** [comment]
Big +1
Can confirm that following it is a lot of fun

**Comment by betatim** [comment]
[Twitter delivers](https://twitter.com/cplberry/status/698061955836162049): [cplberry](https://twitter.com/cplberry) might be able to hook us up. However he asks, what in particular would we want to learn? I'd be happy to hear the inside scoop/assumptions and gory details of how one of the plots in the PRL were made. Other thoughts?

**Comment by tpmccauley** [comment]
Myself, I was thinking of making some graphics or animations with WebGL. Not sure I need to delve too deeply into the analysis for that but at least I have access to some numbers.

**Comment by RaoOfPhysics** [comment]
We should get @cranmer in on the action:
- https://twitter.com/KyleCranmer/status/698210825388032000
  - https://github.com/cranmer/ligo-binder
- https://twitter.com/KyleCranmer/status/698240530900193282

**Comment by lukasheinrich** [comment]
i'd be in on it.

**Comment by RaoOfPhysics** [comment]
:D

**Comment by RaoOfPhysics** [comment]
For reference, via @betatim: https://github.com/minrk/ligo-binder

**Comment by betatim** [comment]
We should figure out what we want to learn and then get in touch with the
LIGO guy on twitter, see if he can connect us to someone with the needed
knowledge

**Comment by RaoOfPhysics** [comment]
I nominate @betatim to lead operations! :D

**Comment by lukasheinrich** [comment]
reproducing this chain in filtering out the signal looks fun:

https://twitter.com/danhoak/status/698448909006405632

not sure if LIGO release all the data to do that, though

**Comment by eamonnmag** [comment]
I also know Jenne Drigger who was at CERN at the same time as me as a summer student. Now she's a post doc on LIGO. Want me to contact her?

**Comment by betatim** [comment]
:thumbsup:

**Comment by RaoOfPhysics** [comment]
@eamonnmag: Any update on this?

**Comment by eamonnmag** [comment]
Yes, @betatim and I were speaking with Jenne. She is happy to support remotely.

**Comment by RaoOfPhysics** [comment]
Woo! Is she on GitHub? If so, perhaps you can add her to this conversation. And maybe we should take a poll on what the best day is to do this, both for Jenne and for the interested folk at CERN.

**Comment by eamonnmag** [comment]
She's not. But if we get a doodle poll together, I can send her that.

**Comment by hepstr** [comment]
@cjhaster (postdoc with LIGO) might also be interested in providing some (remote) help, time permitting.

**Comment by cjhaster** [comment]
Hi everyone, I realised I never said anything about this but as Stefan mentioned I'd be happy to help with this.
I'm primarily working in the parameter estimation group (within the larger compact binary coalescence group) of LIGO, so my day to day work includes passing any interesting data through our (very nice in my opinion) Bayesian parameter estimation pipeline, and especially making this pipeline produce as much physics in the least amount of time possible.
We started our second observing run yesterday (Nov 30), so once any results from that has been published you'll have even more data to play around with  :)
